lib/Remotery.h

```diff
@@ -116,6 +116,12 @@ documented just below this comment.
 #define RMT_USE_INTERNAL_HASH_FUNCTION 1
 #endif
 
+// If RMT_USE_LEGACY_ATOMICS is defined to 1, the implementation will use the legacy fallback atomic functions
+// The default setting is 0
+#ifndef RMT_USE_LEGACY_ATOMICS
+#define RMT_USE_LEGACY_ATOMICS 0
+#endif
```

Comment:
New define:
* is the name ok?
* is the usage ok? i.e. setting/comparing a value, which is a bit different from other defines used.

Reply:
This is exactly how we do stuff like `RMT_USE_D3D12` so good.